Solutions/Vectra AI Detect/Analytic Rules/VectraDetect-Account-Detections.yaml

```diff
@@ -59,7 +59,7 @@ query: |
   | extend Severity = case(level == "Info", "Informational",
       level == "Low", "Low",
       level == "Medium", "Medium",
-      level == "High", "Medium",
+      level == "High", "High",
       level == "Critical", "High",
```

Comment:
Entire block of this case statement can be improvement by using the following:
case(level == "Info", "Informational",
      level == "Critical", "High",
      level)

Reply:
implemented